integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -347,7 +347,7 @@ object CommonUtil {
       sys.error("The size of new list must be smaller than original list, please check again!")
     }
     val tempList = newListInfo.mkString(",").split(",")
```

Comment:
Is it necessary to do `list.mkString.split`?

Reply:
Actually, the List on which these functions are being performed stores values as a List[String]

Suppose the list has the following values: ('Korea', 'China', '(Japan, US, Africa)' , 'UK')
Without applying any function the list elements will be :
list(0) => Korea
list(1) => China
list(2) => (Japan, US, Africa)		//This list of elements is stored like a string
list(3) => UK

But When we perform list.mkString(",").split(",")
The list now have every element as individual value
list(0) => Korea
list(1) => China
list(2) => (Japan
list(3) => US
list(4) => Africa)
list(5) => UK

Reply:
thanks, learned from this... :+1: